src/launcher/standalone_launcher/.dockerignore

```diff
@@ -0,0 +1,4 @@
+
+node_modules
+npm-debug.log
+Dockerfile
```

Comment:
Add newline at end of file. (Please set up your IDE to insert these automatically).